designs/osc-packaging/kolla/osc-kolla.md

```diff
@@ -0,0 +1,70 @@
+# OSC Kolla
+[Kolla](#kolla-home) is an OpenStack project that provides production-ready containers and deployment tools for operating OpenStack clouds. This document outlines the contributions to be up streamed to Kolla so that the OSC image is present in the [Kolla container repository](#kolla-container-repos).   
```

Comment:
Do we have any process to upload OSC kolla container image to kolla repository? how is this done? any procedure to follow to get OSC repository place-holder? do we need to work with the kolla community for putting the osc container image?

Reply:
Yes to all those questions. I have proposed more user stories to track that. But before we involve the community etc we will need to determine whether we will use Kolla (there is also a user story for that). The main purpose of this initial work was to parse the ODL material, learn how OOO deals with basic configuration and identify the next steps/areas for investigation.